project/README.md

```diff
@@ -1,12 +1,31 @@
+## You are here
 Main location for reproducibility study.
 
-Layout is as below (subject to change):
+## How to use
+All commands are intended to be run from this location with the `mosdef-study38` environment active.
+To initialize the project workspace:
+```bash
+python init.py
+```
+You will see the empty workspace folder populated with the hashed job directories.
+
+Each engine directory in `src/engines` contains its own `project.py` file. To run the simulations for a specific engine (in the following example we use hoomd), you would run:
+```bash
```

Comment:
I think we should first show using the `status` command first.

Reply:
good point! I guess we don't know if users will use submit or run anyway...
I'll also link to the signac cli docs